tests/testutil/testutil.go

```diff
@@ -55,6 +55,22 @@ func KubectlCmd(args []string) (string, error) {
 	return string(stdOutErr), nil
 }
 
+type waitForMe = func() bool
```

Comment:
This has been updated yesterday, we are using k8's wait method now: https://github.com/rancher/rio/blob/master/tests/testutil/externalservice.go#L79-L90

So you can remove this.